### ISSUE TITLE
Remove upper cases in import package names

### ISSUE THREAD

**Issue by lumtis** [zeta-chain/node#908]
Package names in imports should be conventionally in full lower case to distinguish from variable names.

```
emissionsModule "github.com/zeta-chain/zetacore/x/emissions"
```
-->
```
emissionsmodule "github.com/zeta-chain/zetacore/x/emissions"
```

I think there might be also package names that can be simplified:
`zetaobservermodulekeeper` --> `observerkeeper`

**Comment by aks3213** [comment]
Hello @lumtis, I can take this one up.

**Comment by fadeev** [comment]
Awesome, @aks3213 🙌 assigning the issue to you. Let us know if you have any questions!

**Comment by aks3213** [comment]
QQ: Only packages named in issue above need to be changed or all similar package alias also need to be changed?

**Comment by lumtis** [comment]
@aks3213 great to hear

I would maybe advise waiting on https://github.com/zeta-chain/node/pull/907 which should be merged very soon or you might have several conflicts to resolve in your branch.

Regarding your question, so far I just noticed that `zetaObserver` prefix is often used for imports of observer module which can be simplified into `observer`.

Otherwise, the general convention in Cosmos SDK to name import package:
```
import (
  ...

  [moduleName] ".../[moduleName]"
  [moduleName]types ".../[moduleName]/types"
  [moduleName]keeper ".../[moduleName]/keeper"
)
```

**Comment by lumtis** [comment]
For example in the issue description
```
emissionsmodule "github.com/zeta-chain/zetacore/x/emissions"
```
can actually be:
```
emissions "github.com/zeta-chain/zetacore/x/emissions"
```

**Comment by aks3213** [comment]
Understood. Thanks @lumtis 
Seems [#907](https://github.com/zeta-chain/node/pull/907) is merged now.
Can I start with this one now?

**Comment by lumtis** [comment]
> Understood. Thanks @lumtis Seems [#907](https://github.com/zeta-chain/node/pull/907) is merged now. Can I start with this one now?

Yes it was merged.

**Comment by aks3213** [comment]
I have changed import alias as described above. Now I am following this [contrib\localnet\README.md](https://github.com/aks3213/node/blob/develop/contrib/localnet/README.md) file to test my changes to make sure everything is okay. 
I followed these steps in root folder of repository.

- make zetanode
```fatal: No names found, cannot describe anything.
Building zetanode
docker build -t zetanode -f ./Dockerfile .
[+] Building 2.6s (30/30) FINISHED                                                                                                                                            
 => [internal] load build definition from Dockerfile                                                                                                                     0.0s
 => => transferring dockerfile: 1.31kB                                                                                                                                   0.0s 
 => [internal] load .dockerignore                                                                                                                                        0.1s 
 => => transferring context: 95B                                                                                                                                         0.0s 
 => [internal] load metadata for docker.io/library/golang:1.19-alpine                                                                                                    2.4s
 => [stage-0  1/25] FROM docker.io/library/golang:1.19-alpine@sha256:4365a83ba4680d5367167ce255b01a207c92ddaf4de0964d29a3cc63d1d9324e                                    0.0s
 => [internal] load build context                                                                                                                                        0.2s 
 => => transferring context: 52.82kB                                                                                                                                     0.2s 
 => CACHED [stage-0  2/25] RUN apk --no-cache add git make build-base jq openssh libusb-dev linux-headers bash curl tmux                                                 0.0s
 => CACHED [stage-0  3/25] RUN ssh-keygen -b 2048 -t rsa -f /root/.ssh/localtest.pem -q -N ""                                                                            0.0s 
 => CACHED [stage-0  4/25] WORKDIR /go/delivery/zeta-node                                                                                                                0.0s 
 => CACHED [stage-0  5/25] COPY go.mod .                                                                                                                                 0.0s 
 => CACHED [stage-0  6/25] COPY go.sum .                                                                                                                                 0.0s 
 => CACHED [stage-0  7/25] RUN --mount=type=cache,target=/root/.cache/go-build     go mod download                                                                       0.0s 
 => CACHED [stage-0  8/25] COPY . .                                                                                                                                      0.0s 
 => CACHED [stage-0  9/25] RUN --mount=type=cache,target=/root/.cache/go-build     make install                                                                          0.0s 
 => CACHED [stage-0 10/25] RUN --mount=type=cache,target=/root/.cache/go-build     make install-smoketest                                                                0.0s 
 => CACHED [stage-0 11/25] RUN ssh-keygen -A                                                                                                                             0.0s 
 => CACHED [stage-0 12/25] WORKDIR /root                                                                                                                                 0.0s 
 => CACHED [stage-0 13/25] RUN cp /root/.ssh/localtest.pem.pub /root/.ssh/authorized_keys                                                                                0.0s 
 => CACHED [stage-0 14/25] RUN cp /go/bin/zetaclientd /usr/local/bin                                                                                                     0.0s 
 => CACHED [stage-0 15/25] RUN cp /go/bin/zetacored /usr/local/bin                                                                                                       0.0s 
 => CACHED [stage-0 16/25] RUN cp /go/bin/smoketest /usr/local/bin                                                                                                       0.0s 
 => CACHED [stage-0 17/25] COPY contrib/localnet/scripts /root                                                                                                           0.0s 
 => CACHED [stage-0 18/25] COPY contrib/localnet/preparams /root/preparams                                                                                               0.0s 
 => CACHED [stage-0 19/25] COPY contrib/localnet/ssh_config /root/.ssh/config                                                                                            0.0s 
 => CACHED [stage-0 20/25] COPY contrib/localnet/zetacored /root/zetacored                                                                                               0.0s 
 => CACHED [stage-0 21/25] COPY contrib/localnet/tss /root/tss                                                                                                           0.0s 
 => CACHED [stage-0 22/25] RUN chmod 755 /root/*.sh                                                                                                                      0.0s 
 => CACHED [stage-0 23/25] RUN chmod 700 /root/.ssh                                                                                                                      0.0s 
 => CACHED [stage-0 24/25] RUN chmod 600 /root/.ssh/*                                                                                                                    0.0s 
 => CACHED [stage-0 25/25] WORKDIR /usr/local/bin                                                                                                                        0.0s 
 => exporting to image                                                                                                                                                   0.0s 
 => => exporting layers                                                                                                                                                  0.0s 
 => => writing image sha256:3ad0fedd66f27ce617057ef5b53f26ba136db0fde7ce729df01a71afb7ddd0ad                                                                             0.0s 
 => => naming to docker.io/library/zetanode                                                                                                                              0.0s 
docker build -t orchestrator -f contrib/localnet/orchestrator/Dockerfile.fastbuild .
[+] Building 2.1s (21/21) FINISHED
 => [internal] load .dockerignore                                                                                                                                        0.0s
 => => transferring context: 95B                                                                                                                                         0.0s 
 => [internal] load build definition from Dockerfile.fastbuild                                                                                                           0.0s 
 => => transferring dockerfile: 743B                                                                                                                                     0.0s 
 => [internal] load metadata for docker.io/library/golang:1.19-alpine                                                                                                    0.4s
 => [internal] load metadata for docker.io/library/zetanode:latest                                                                                                       0.0s 
 => [internal] load metadata for docker.io/ethereum/client-go:v1.10.26                                                                                                   1.9s 
 => [zeta 1/1] FROM docker.io/library/zetanode:latest                                                                                                                    0.0s
 => [geth 1/1] FROM docker.io/ethereum/client-go:v1.10.26@sha256:03604c12f6123fda67f534c9a68a73dbabd35d5c0a13f0c6cfcf735cfd8760c8                                        0.0s 
 => [internal] load build context                                                                                                                                        0.1s 
 => => transferring context: 172B                                                                                                                                        0.0s 
 => [orchestrator  1/12] FROM docker.io/library/golang:1.19-alpine@sha256:4365a83ba4680d5367167ce255b01a207c92ddaf4de0964d29a3cc63d1d9324e                               0.0s 
 => CACHED [orchestrator  2/12] RUN apk --no-cache add jq curl bash make git build-base                                                                                  0.0s 
 => CACHED [orchestrator  3/12] COPY --from=zeta /usr/local/bin/zetacored /usr/local/bin/                                                                                0.0s 
 => CACHED [orchestrator  4/12] COPY --from=zeta /usr/local/bin/zetaclientd /usr/local/bin/                                                                              0.0s 
 => CACHED [orchestrator  5/12] COPY --from=geth /usr/local/bin/geth /usr/local/bin/                                                                                     0.0s 
 => CACHED [orchestrator  6/12] COPY --from=zeta /root/.ssh/localtest.pem.pub /root/.ssh/authorized_keys                                                                 0.0s 
 => CACHED [orchestrator  7/12] COPY --from=zeta /root/.ssh/localtest.pem /root/.ssh/localtest.pem                                                                       0.0s 
 => CACHED [orchestrator  8/12] COPY contrib/localnet/orchestrator/start.sh /work/                                                                                       0.0s 
 => CACHED [orchestrator  9/12] RUN chmod +x /work/start.sh                                                                                                              0.0s 
 => CACHED [orchestrator 10/12] COPY --from=zeta /usr/local/bin/smoketest /usr/local/bin/                                                                                0.0s 
 => CACHED [orchestrator 11/12] RUN chmod +x /usr/local/bin/smoketest                                                                                                    0.0s 
 => CACHED [orchestrator 12/12] WORKDIR /work                                                                                                                            0.0s 
 => exporting to image                                                                                                                                                   0.0s 
 => => exporting layers                                                                                                                                                  0.0s 
 => => writing image sha256:6885ebd53d17981bd5c6b776a83def03fcb3cd1409fc9f4da4b4fa49ee61bf47                                                                             0.0s 
 => => naming to docker.io/library/orchestrator
```

- docker build -t zetanode .
```
[+] Building 1.8s (30/30) FINISHED
 => [internal] load .dockerignore                                                                                                                                        0.0s
 => => transferring context: 95B                                                                                                                                         0.0s 
 => [internal] load build definition from Dockerfile                                                                                                                     0.0s 
 => => transferring dockerfile: 1.31kB                                                                                                                                   0.0s 
 => [internal] load metadata for docker.io/library/golang:1.19-alpine                                                                                                    0.9s 
 => [internal] load build context                                                                                                                                        0.7s
 => => transferring context: 12.27MB                                                                                                                                     0.7s
 => [stage-0  1/25] FROM docker.io/library/golang:1.19-alpine@sha256:4365a83ba4680d5367167ce255b01a207c92ddaf4de0964d29a3cc63d1d9324e                                    0.0s 
 => CACHED [stage-0  2/25] RUN apk --no-cache add git make build-base jq openssh libusb-dev linux-headers bash curl tmux                                                 0.0s
 => CACHED [stage-0  3/25] RUN ssh-keygen -b 2048 -t rsa -f /root/.ssh/localtest.pem -q -N ""                                                                            0.0s 
 => CACHED [stage-0  4/25] WORKDIR /go/delivery/zeta-node                                                                                                                0.0s 
 => CACHED [stage-0  5/25] COPY go.mod .                                                                                                                                 0.0s 
 => CACHED [stage-0  6/25] COPY go.sum .                                                                                                                                 0.0s 
 => CACHED [stage-0  7/25] RUN --mount=type=cache,target=/root/.cache/go-build     go mod download                                                                       0.0s 
 => CACHED [stage-0  8/25] COPY . .                                                                                                                                      0.0s
 => CACHED [stage-0  9/25] RUN --mount=type=cache,target=/root/.cache/go-build     make install                                                                          0.0s 
 => CACHED [stage-0 10/25] RUN --mount=type=cache,target=/root/.cache/go-build     make install-smoketest                                                                0.0s 
 => CACHED [stage-0 11/25] RUN ssh-keygen -A                                                                                                                             0.0s 
 => CACHED [stage-0 12/25] WORKDIR /root                                                                                                                                 0.0s 
 => CACHED [stage-0 13/25] RUN cp /root/.ssh/localtest.pem.pub /root/.ssh/authorized_keys                                                                                0.0s 
 => CACHED [stage-0 14/25] RUN cp /go/bin/zetaclientd /usr/local/bin                                                                                                     0.0s 
 => CACHED [stage-0 15/25] RUN cp /go/bin/zetacored /usr/local/bin                                                                                                       0.0s 
 => CACHED [stage-0 16/25] RUN cp /go/bin/smoketest /usr/local/bin                                                                                                       0.0s 
 => CACHED [stage-0 17/25] COPY contrib/localnet/scripts /root                                                                                                           0.0s 
 => CACHED [stage-0 18/25] COPY contrib/localnet/preparams /root/preparams                                                                                               0.0s 
 => CACHED [stage-0 19/25] COPY contrib/localnet/ssh_config /root/.ssh/config                                                                                            0.0s 
 => CACHED [stage-0 20/25] COPY contrib/localnet/zetacored /root/zetacored                                                                                               0.0s 
 => CACHED [stage-0 21/25] COPY contrib/localnet/tss /root/tss                                                                                                           0.0s 
 => CACHED [stage-0 22/25] RUN chmod 755 /root/*.sh                                                                                                                      0.0s 
 => CACHED [stage-0 23/25] RUN chmod 700 /root/.ssh                                                                                                                      0.0s
 => CACHED [stage-0 24/25] RUN chmod 600 /root/.ssh/*                                                                                                                    0.0s 
 => CACHED [stage-0 25/25] WORKDIR /usr/local/bin                                                                                                                        0.0s 
 => exporting to image                                                                                                                                                   0.0s 
 => => exporting layers                                                                                                                                                  0.0s 
 => => writing image sha256:3ad0fedd66f27ce617057ef5b53f26ba136db0fde7ce729df01a71afb7ddd0ad                                                                             0.0s 
 => => naming to docker.io/library/zetanode
```

- make start-smoketest
```
fatal: No names found, cannot describe anything.
--> Starting smoketest
cd contrib/localnet/ && docker compose up -d
[+] Running 10/10
 ✔ bitcoin 5 layers [⣿⣿⣿⣿⣿]      0B/0B      Pulled                                                                                                                      39.9s 
   ✔ 3f9582a2cbe7 Pull complete                                                                                                                                         34.9s 
   ✔ 23a697d6731f Pull complete                                                                                                                                         35.4s 
   ✔ f8ee33a99101 Pull complete                                                                                                                                         35.8s 
   ✔ c45a487d8241 Pull complete                                                                                                                                         36.0s 
   ✔ 4f4fb700ef54 Pull complete                                                                                                                                         36.1s 
 ✔ eth 3 layers [⣿⣿⣿]      0B/0B      Pulled                                                                                                                             4.6s 
   ✔ 213ec9aee27d Already exists                                                                                                                                         0.0s 
   ✔ 4a36b71a98b2 Already exists                                                                                                                                         0.0s 
   ✔ ee12b8715f0d Already exists                                                                                                                                         0.0s 
[+] Running 9/9
 ✔ Network localnet_mynetwork  Created                                                                                                                                   0.1s 
 ✔ Container zetaclient1       Started                                                                                                                                  13.0s 
 ✔ Container eth               Started                                                                                                                                  15.6s 
 ✔ Container zetacore1         Started                                                                                                                                  12.7s 
 ✔ Container zetaclient0       Started                                                                                                                                  15.5s 
 ✔ Container zetacore0         Started                                                                                                                                  16.0s 
 ✔ Container rosetta           Started                                                                                                                                  15.5s 
 ✔ Container bitcoin           Started                                                                                                                                  16.0s 
 ✔ Container orchestrator      Started    
```

Now when I go to docker I see only below.
![image](https://github.com/zeta-chain/node/assets/68645565/01f5d724-8112-47ad-842c-7e9795fc7306)

I am not able to see "smoketest done" log.
Please guide me if I am doing anything wrong here.

**Comment by lumtis** [comment]
Hey @aks3213 , if the smoke test runs without error and doesn't stop it is good.
The doc may need to be updated.

**Comment by aks3213** [comment]
Please review this https://github.com/zeta-chain/node/pull/917
Let me know if anything needs to be updated in description of PR.

**Comment by aks3213** [comment]
raised a new PR for this one https://github.com/zeta-chain/node/pull/1001
plz review

**Comment by lumtis** [comment]
Issue mostly completed in the codebase. We can remove eventual remaining one in other PRs
